project/templates/unicoremc/home.html

```diff
@@ -14,48 +18,96 @@
+    <div class="row">
+        <div class="nav-tabs-custom main col-xs-2 card new-site">
+            <div class="card-content new-site">
+                <div id="overlay-content" class="card-content-overlay">
+                    {% if user.is_superuser %}
+                        <div>
+                            <h4>Choose website type</h4>
+                            <hr/>
+                            <a href="{% url 'base:add' %}" class="btn btn-primary">Basic</a>
+                            <a href="{% url 'controllers.docker:add' %}" class="btn btn-primary">Docker</a>
+                            <a href="{% url 'controllers.freebasics:add' %}" class="btn btn-primary">
+                                <span class="free-basics"><i>&ensp;&ensp;&ensp;&ensp;</i></span>Free Basics
+                            </a>
+                        </div>
+                    {% else %}
+                        <div class="center-text">
+                            <span><i class="fa fa-times-circle fa-5x"></i></span><h3>You do not have permission to create a new website</h3>
+                        </div>
+                    {% endif %}
+                </div>
+                <div id="overlay-title" class="card-content-overlay">
+                    <div class="center-text">
+                        <h1>Create Mobi Site</h1>
+                        <hr/>
+                        <h3>Setup and create a new mobile site</h3>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="nav-tabs-custom main col-xs-2 card hosting">
+            Hosting
+        </div>
+    </div>
+    <div class="row">
+        <h3 class="title-text">Manage your applications</h3>
+    </div>
+    <div class="row" id="result_list">
         {% for controller in controller_list %}
         {% render_controller controller %}
         {% endfor %}
-  </div>
-  <div class="col-md-3 col-sm-6 col-xs-12">
-    <div class="info-box">
-      <span class="info-box-icon bg-green"><i class="fa fa-flag-o"></i></span>
-
-      <div class="info-box-content">
-        <span class="info-box-text">Applications</span>
-        <span class="info-box-number">{{controller_list.count}}</span>
-        <div class="progress">
-          <div class="progress-bar progress-bar-green" style="width: 100%"></div>
-        </div>
-        <span class="progress-description">
-          Unlimitted
-        </span>
-      </div>
     </div>
-  </div>
-  <div class="col-md-3 col-sm-6 col-xs-12">
-    <div class="info-box">
-      <span class="info-box-icon bg-yellow"><i class="fa fa-dollar"></i></span>
-      <div class="info-box-content">
-        <span class="info-box-text">Hosting</span>
-        <span class="info-box-number">${{controller_list.count|multiply:25}}</span>
-        <span class="progress-description">
-          $25 per app
-        </span>
-      </div>
+{#  <div class="col-md-3 col-sm-6 col-xs-12">#}
```

Comment:
Instead of commenting out, you can just delete it. Github has the history if we need to look back.